src/components/Project.jsx

```diff
@@ -0,0 +1,144 @@
+    };
+
+    const IsNotHovered = () => {
+        // console.log("no Hover");
+    };
+
+    useEffect(() => {
+        // update the classes and effects based on the state
+        switch (_is_being_hovered_) {
+            case true:
+                IsHovered();
+                break;
+            case false:
+                IsNotHovered();
+        }
+    }, [_is_being_hovered_]);
+
+    // when the project is hovered over
+    const hoverCallBack = () => {
+        setState(true);
+    };
+
+    const hoverLeft = () => {
+        setState(false);
+    }
+
+    return (
+        <div {...props} ref={ref} className={` GNOMEproject_div_pos `}>
+            <div className={commonDivClass + moreDetailsClass} style={moreDetailsStyle}>
+                <h3 className={headerClass} style={headerStyle}>More Details</h3>
+                <p>
+                    {details_2}
+                </p>
+            </div>
+            <div onMouseOver={hoverCallBack} onMouseLeave={hoverLeft} style={mainDivStyle} className={commonDivClass + mainDivClass}>
+                <Image src={link_to_img} className={projImgClass + (_is_being_hovered_ ? "" : " ")} style={projImgStyle} />
+                <div className={contributorsClass + (_is_being_hovered_ ? " GNOMEContributorClass " : " GNOMEHide")} style={contributorsStyle}>
+                    <h5 className={headerClass} style={headerStyle}>
+                        {project_name}
+                        <p style={{ width: "0", height: "0" }} className={(_is_being_hovered_ ? " GNOMEUnderline-effect" : "")}></p>
+                    </h5>
+                    <h5>
+                        <button className={buttonClass} style={buttonStyle}><a style={linkStyle} href={github_link} className={linkClass}>Go to GitHub</a></button>
+                        <button className={buttonClass} style={buttonStyle}><a style={linkStyle} href={path_to_activate} className={linkClass}>Go to Website</a></button>
+                    </h5>
+                    <h5 className={headerClass} style={headerStyle}>
+                        Contributors
+                        <p style={{ width: "0", height: "0" }} className={(_is_being_hovered_ ? " GNOMEUnderline-effect" : "")}></p>
+                    </h5>
+                    <div className="flex flex-row flex-wrap px-4">
+                        {contributors.map((ent, ind) => {
+                            return (<Image key={ind} src={fetchContributor(ent).link_to_img} className={ind.toString() + contsPfpClass + (_is_being_hovered_ ? " GNOMERush_up " : "")} style={contsPfpStyle} />);
+                        })}
+                    </div>
+                </div>
+            </div>
+            <div className={commonDivClass + detailsClass} style={detailsStyle}>
+                <h3 className={headerClass} style={headerStyle} >Details</h3>
+                <p>
+                    {details_1}
+                </p>
+            </div>
+        </div>
+    );
+});
+{/*
+*/
+}
+
```

Comment:
Use `prettier` to format these codes. You can install `prettier` extension in your VS Code and format it manually. [Read more about it](https://prettier.io/docs/en/editors)